docs/workflow/issue-management.md

```diff
@@ -0,0 +1,61 @@
+# Issue management
+
+- [How to create an issue](#how-to-create-an-issue)
+- [How to pick an issue](#how-to-pick-an-issue)
+
+## How to create an issue
+
+1. Go to `Issues` tab in the repository and click on the green `New issue` button
+
+2. Fill out the issue template:
+    
+    2.1. `Dependencies` - if the issue is waiting on another issue
+
+    2.2. `Problem` - what are we solving
+
+    2.3. `Solution suggestion` - suggest a solution
+
+    2.4. `Acceptance criteria` - issue outcomes, what have we achieved?
+
+    2.5. `Learning outcome` - how did we benefit from it personally, what did we learn?
+
+3. Add relevant labels:
+
+    3.1. Estimate how long the issue is going to take (1h / 2h / 3h)
+
+        If it takes more than 3 hours, then it should be split (e.g. into research and actual implementation)
+
```

Comment:
I also realized that we can estimate not only issues but whole "milestones" which are like collections of issues, based on our weekly availability. We should try and divide the whole project into milestones ahead of time then.